### PR TITLE
[Feature] Search People

### PR DESCRIPTION
**Pull request by raphaelsattler**
Create search people feature and its tests

### CODE REVIEW

File: app/views/people/index.html.erb [hunk 1]
```diff
@@ -1 +1,43 @@
...
+        <%= f.search_field :cpf_cont, class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <%= f.label :email_cont, t(:email) %>
+        <%= f.search_field :email_cont, class: "form-control" %>
+      </div>
+      <div class="d-flex justify-content-around">
+        <div>
+        <%= f.label :profile_name_cont, t(:profile) %>
+          <%= f.collection_select :profile_name_cont, Profile.all, :name, :name, { include_blank: true }, { class: "btn-sm" } %>
```
Comment (RodrigoNM) [inline app/views/people/index.html.erb]: This should be a helper.

File: app/assets/stylesheets/table.scss.css [hunk 2]
```diff
@@ -8,6 +8,8 @@ tr:nth-child(even) {
   background-color: #f2f2f2;
 }
 .table-head {
-  color: #FFFFFF;
   background-color: #6577FF;
 }
+.column-name {
+  color: #FFFFFF !important;
```
Comment (kaiomagalhaes) [inline app/assets/stylesheets/table.scss.css]: Never use !important. If you need it to be more specific then create an id

File: app/controllers/person_controller.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class PersonController < ApplicationController
+  def index
+    @q = Person.ransack(params[:q])
+    @people = @q.result.includes(:profile, :buy_intention).paginate(page: params[:page], per_page: 5)
```
Comment (kaiomagalhaes) [inline app/controllers/person_controller.rb:4]: use the safe params here

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/form_filter.scss.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.form-head {
+  background-color: #6577FF !important;
```
Comment (kaiomagalhaes) [inline app/assets/stylesheets/form_filter.scss.css]: please avoid !Important, instead use `id`

File: app/assets/stylesheets/form_filter.scss.css [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  color: #FFFFFF;
+}
+
+.form-selector {
+  background-color: #6577FF !important;
```
Comment (kaiomagalhaes) [inline app/assets/stylesheets/form_filter.scss.css]: please avoid !Important, instead use id

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

**Review by kaiomagalhaes — APPROVED** [review]

**Review by Heirian — APPROVED** [review]

**Pull request merged**